zilsd.adoc

```diff
@@ -38,11 +38,16 @@ In this case, these instruction are guaranteed to not raise an address-misaligne
 Even if naturally aligned, the memory access might not be performed atomically.
 
 If the effective address is a multiple of 4, then each word access is required to be performed atomically.
-The LD instruction must however write the loaded data to the pair of destination registers atomically to ensure fault handling is possible.
+
+To ensure fault handling is possible for the load instructions, it must be ensured that the register which is the source of the base address is not overwritten before the entire operation is complete.
+This affects x2 for the stack pointer relative instruction and rs1 otherwise.
+To guarantee this, if one of the destination registers of the pair is the source register containing the base, it must not be written to before the other register is the pair has been written.
```

Comment:
Typo: "is" instead of "in" (the pair).

Also, this seems a bit redundant (since loads only have one source register):
> To guarantee this, if one of the destination registers of the pair is the source register containing the base, it must not be written to before the other register is the pair has been written.

Maybe something like this instead?
> To guarantee this, if one of the destination registers of the pair is also the source, it must not be written before the other register in the pair.

Reply:
I found just source a bit two ambiguous, particularly for c.ldsp, which does not have an explicit source operand. I am open to any suggestions. Meanwhile, I will fix the typo.